examples/shared/base-ui/src/components/ui/alert-dialog.tsx

```
@@ -1,7 +1,7 @@
 'use client';
 
-import { buttonVariants } from '@examples/design-system/components/ui/button';
-import { cn } from '@examples/design-system/lib/utils';
+import { buttonVariants } from '@examples/base-ui/components/ui/button';
```

ESLint

[error] Unable to resolve path to module '@examples/base-ui/components/ui/button'.

(import-x/no-unresolved)